### DISCUSSION TITLE
How to install along with tensorflow 2.4.1?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [coqui-ai/STT#1801] [Q&A]
I have tensorflow 2.4.1 installed, but when run

```
python setup.py install
```

I got this:

```
Searching for tensorflow==1.15.4
Reading https://pypi.org/simple/tensorflow/
No local packages or working download links found for tensorflow==1.15.4
error: Could not find suitable distribution for Requirement.parse('tensorflow==1.15.4')

```

why it force a version? And I can not found where this requirements come from so that nowhere to comment it.

**Comment by erogol** [comment]
STT is not officially ready for TF >=2.0 since our low level inference graph is not readily compatible with it, but it is WIP.

**Reply by reuben** [reply]
Note that this only applies to the training code. The inference packages are built with TensorFlow 2.3 (but you don't have to install it separately anyway).

**Reply by lucasjinreal** [reply]
@reuben How do I install inference along since I don't want using tensorflow 1.15 anymore. I dont want reinstall my tensorflow. I just want ifnerence

**Reply by reuben** [reply]
@jinfagang follow the docs: https://stt.readthedocs.io/en/latest/DEPLOYMENT.html
